src/components/widget-creator.jsx

```diff
@@ -103,14 +103,14 @@ const WidgetCreator = ({instId, widgetId, minHeight='', minWidth=''}) => {
 		placeholderData: null,
 		enabled: !!instance.id, // requires instance state object to be prepopulated
 		onSettled: (data) => {
-			if ( (data != null ? data.title : undefined) === 'Permission Denied' ||data.title === 'error') {
-					setCreatorState({...creatorState, invalid: true})
-					onInitFail('Permission Denied')
-				} else {
-					setCreatorState({...creatorState, invalid: false})
-					setInstance({ ...instance, qset: data })
-				}
+			if ( (data != null ? data.title : undefined) === 'Permission Denied' || (data && data.title === 'error')) {
```

Comment:
This check won't catch `data == null`, but the API should never send a null object for this request. If an instance qset isn't found it sends a dummy one instead, so this should be sufficient.